pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -127,14 +127,10 @@ rec {
         ]
       ];
 
-      # When cross-compiling checks need to be enabled explicitly via crossAttrs
-      doCheck' = if stdenv.hostPlatform == stdenv.buildPlatform
-                 then doCheck
-                 else attrs.crossAttrs.doCheck or false;
-
-      doInstallCheck' = if stdenv.hostPlatform == stdenv.buildPlatform
-                        then doInstallCheck
-                        else attrs.crossAttrs.doInstallCheck or false;
+      # TODO(@oxij, @Ericson2314): This is here to keep the old semantics, remove when
+      # no package has `doCheck = true`.
```

Comment:
"no package has `doCheck = true`." is an excellent way to phrase it, thanks :).

Reply:
:)